coreax/util.py

```diff
@@ -201,27 +202,39 @@ def solve_qp(kernel_mm: ArrayLike, kernel_matrix_row_sum_mean: ArrayLike) -> Arr
     return sol.primal
 
 
-def jit_test(fn: Callable, *args, **kwargs) -> tuple[float, float]:
+def jit_test(
+    fn: Callable,
+    fn_args: tuple = (),
+    fn_kwargs: dict | None = None,
```

Comment:
`fn_args`, `fn_kwargs` and `jit_kwargs` need adding to the docstring with an appropriate description